app/main.js

```diff
@@ -915,6 +923,47 @@ function showNotification (text) {
   })
 }
 
+function runCommand (event, argv, workingDirectory) {
+  const args = argv.slice(app.isPackaged ? 1 : 2)
+  const cmd = new Command(args, app.getVersion())
+
+  // if this command is already executed by the second-instance, return early
+  if (!cmd.checkInMain()) return
+
+  switch (cmd.command) {
+    case 'reset':
+      resetBreaks()
+      break
+
+    case 'mini':
+      if (cmd.options.title) nextIdea = [cmd.options.title, '']
+      if (cmd.options.skip) skipToMicrobreak(false)
```

Comment:
`skipToBreak` and `skipToMicrobreak` don't have any params

Reply:
https://github.com/papey/stretchly/commit/470ba06faafad669674b590b48f08e5ae0e504d8